untitled/src/homework2/task3.java

```diff
@@ -0,0 +1,15 @@
+package homework2;
+
+public class task3 {
+    public static void main(String[] args) {
+        boolean shouldContinue = true;
```

Comment:
Not wrong. In this ,case you could do it with for loop - will be easy

        for(int i = -100 ; i < 101 ; i++) {
            System.out.println(i);
        }